platform/ios/src/MGLAnnotationView.h

```diff
@@ -41,6 +41,19 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, assign, getter=isFlat) BOOL flat;
 
 /**
+ This property defaults to NO and becomes YES when the view is tapped/clicked on.
```

Comment:
Nit: Let’s remove the word “clicked”; that term is only appropriate for OS X, and this file isn’t shared with the OS X SDK.